scene_graph/vrd_tutorial.py

```diff
@@ -26,38 +41,45 @@
 # - `source_img`: filename for the corresponding image the relationship is in
 # - `subject_bbox`: coordinates of the bounding box for the object `[ymin, ymax, xmin, xmax]`
 # - `subject_category`: category of the subject
-#
-# Note that the training DataFrame will have a labels field with all -1s. This denotes the lack of labels for that particular dataset. In this tutorial, we will assign probabilistic labels to the training set by writing labeling functions over attributes of the subject and objects!
 
-# %%
+# +
 # %load_ext autoreload
 # %autoreload 2
 
 import numpy as np
 
-# %%
+# -
+
+# If you are running this notebook for the first time, it will take ~15 mins to download all the required data. **Setting `sample = False` will download the full VRD dataset, which can take ~3 hours!**
+#
+# The sampled version of the dataset **uses the same 26 examples across the train, dev, and test sets. This setting is meant to demonstrate how Snorkel works with this task, NOT TO DEMONSTRATE PERFORMANCE.**
```

Comment:
nit: can we use less bold? bold + caps seems like a lot :)
maybe just italicize the caps part?